docs/src/test/scala/docs/home/actor/ActorServiceSpec.scala

```diff
@@ -16,8 +16,8 @@ import java.util.concurrent.TimeUnit
 object ActorServiceSpec {
   def config = ConfigFactory.parseString("""
     akka.actor.provider = akka.cluster.ClusterActorRefProvider
-    akka.remote.netty.tcp.port = 0
-    akka.remote.netty.tcp.hostname = 127.0.0.1
+    akka.remote.artery.canonical.port = 0
+    akka.remote.artery.canonical.hostname = 127.0.0.1
```

Comment:
This is actually part of #1904. It is necessary there since the test is failing. There are other places we need to do a similar chance.

Reply:
Otherwise you would have to disable artery and rename any classic remoting config, such as this one.
In my opinion it’s better/easier to just switch to artery in this PR.

Reply:
Here are more changes that are needed (I think I copied wrong before): https://github.com/lagom/lagom/commit/583127ad599f7b0232fd114a2ae575e111602017

Reply:
@patriknw I made a similar change in #1904, but yours is more comprehensive. I will handle these changes there since I think it is the correct place. We are seeing them here because this PR is built on top of #1904.

Reply:
Now that #1904 was merged, see #1965 for some additional fixes that @patriknw did in the commit above.